internal/config/config.go

```diff
@@ -114,6 +116,14 @@ func extractVersionAnalyzersArguments(cmd *cobra.Command) {
 				logrus.Errorf("couldn't add %v flag: %v", argNamePrefix+argDefinition.Name, err.Error())
 				continue
 			}
+
+			envKey := strings.ToUpper(argNamePrefix + argDefinition.Name)
+			envKey = strings.ReplaceAll(envKey, "-", "_")
+			envKey = strings.ReplaceAll(envKey, ".", "_")
+			envValue, ok := os.LookupEnv(EnvsPrefix + "_" + envKey)
+			if ok {
+				argValue = envValue
```

Comment:
This overrides the `--flag`; but IMO the `--flag`s should override the `SEMANGIT_FLAG`s, since they're more explicit.